## Copy Number Variation callers
These mostly try to measure change in depth due to amplification.

|caller|orig pub|from|study|source|
|------|--------|----|-----|------|
|[bic-seq2](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#bic-seq2)|2016|Harvard, Park lab|[study](https://academic.oup.com/nar/article/44/13/6274/2457607/Copy-number-analysis-of-whole-genome-data-using)|[source](http://compbio.med.harvard.edu/BIC-seq/)|
|[clamms](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#clamms)|2016|Regeneron Genetics Center|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4681995/)|[source](https://github.com/rgcgithub/clamms)|
|[cnvkit](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvkit)|2016|UCSF|[study](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004873)|[source](http://cnvkit.readthedocs.io/en/stable/index.html)|
|[erds-pe](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#erds-pe)|2016|Harbin Institute of Technology, China|[study](https://www.computer.org/csdl/proceedings/bibm/2016/1611/00/07822508-abs.html)|[source](https://github.com/microtan0902/erds-pe)|
|[ioncopy](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#ioncopy)|2016|Charité University Hospital, Berlin|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914355/)|[source](https://cran.r-project.org/web/packages/ioncopy/index.html)|
|[popsv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#popsv)|2016|McGill U Canada, Bourque lab|[study](http://biorxiv.org/content/early/2016/05/09/034165)|[source](https://github.com/jmonlong/PopSV/blob/master/5-FAQ.md)|
|[srbreak](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#srbreak)|2016|University of Otego, New Zealand|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5023681/)|[source](https://github.com/hoangtn/SRBreak)|
|[triocnv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#triocnv)|2016|Harbin Institute of Technology, China|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4907378/)|[source](https://github.com/yongzhuang/TrioCNV)|
|[as-genseng](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#as-genseng)|2015|UNC Chapel Hill|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4538801/)|[source](https://sourceforge.net/projects/asgenseng/)|
|[codex](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#codex)|2015|U Penn Philadelphia|[study](https://www.ncbi.nlm.nih.gov/pubmed/25618849)|[source](http://www.bioconductor.org/packages/devel/bioc/html/CODEX.html)|
|[conserting](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#conserting)|2015|St Jude|[study](https://www.ncbi.nlm.nih.gov/pubmed/25938371)|[source](http://www.stjuderesearch.org/site/lab/zhang)|
|[copywriter](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#copywriter)|2015|Netherlands Cancer Institute|[study](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0617-1)|[source](https://github.com/PeeperLab/CopywriteR)|
|[falcon](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#falcon)|2015|UC Davis|[study](https://www.ncbi.nlm.nih.gov/pubmed/25477383)|[source](https://github.com/PacificBiosciences/FALCON/wiki/Manual)|
|[grom-rd](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#grom-rd)|2015|Grigoriev-Lab|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4369336)|[source](http://grigoriev.rutgers.edu/software/grom-rd/index.html)|
|[modsara](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#modsara)|2015|Yale, Zhang lab|[study](https://academic.oup.com/bioinformatics/article/31/9/1341/200571/Modified-screening-and-ranking-algorithm-for-copy)|[source](http://c2s2.yale.edu/software/modSaRa/)|
|[abscn-seq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#abscn-seq)|2014|UCSD, Messer|[study](http://bioinformatics.oxfordjournals.org/content/early/2014/01/13/bioinformatics.btt759.long)|[source](http://biostats.mcc.ucsd.edu/files/absCNseq_1.0.tar.gz)|
|[adtex](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#adtex)|2014|U Melbourne, Halgamuge|[study](http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-732)|[source](http://adtex.sourceforge.net/tutorial.html)|
|[canoes](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#canoes)|2014|Columbia U|[study](http://nar.oxfordjournals.org/content/early/2014/04/25/nar.gku345.long)|[source](http://www.columbia.edu/~ys2411/canoes/)|
|[climat](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#climat)|2014|U Hefei, China|[study](https://www.ncbi.nlm.nih.gov/pubmed/24845652)|[source](http://bioinformatics.ustc.edu.cn/CLImAT/)|
|[cnvcapseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvcapseq)|2014|Imperial College London|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4227763/)|[source](https://sourceforge.net/projects/cnvcapseq/)|
|[cnvoffseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvoffseq)|2014|Imperial College London, Coin|[study](http://bioinformatics.oxfordjournals.org/content/30/17/i639.long)|[source](https://sourceforge.net/projects/cnvoffseq/)|
|[cnvrd2](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvrd2)|2014|University of Otego Dunedin, New Zealand|[study](https://www.ncbi.nlm.nih.gov/pubmed/25136349/)|[source](http://www.bioconductor.org/packages/devel/bioc/html/CNVrd2.html)|
|[m-hmm](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#m-hmm)|2014|NHGRI|[study](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-109)|[source](https://www.stt.msu.edu/users/hengwang/mHMM.html)|
|[oncocnv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#oncocnv)|2014|Curie Institute|[study](https://www.ncbi.nlm.nih.gov/pubmed/25016581)|[source](http://boevalab.com/ONCOCNV/)|
|[patterncnv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#patterncnv)|2014|Mayo|[study](https://bioinformatics.oxfordjournals.org/content/30/18/2678)|[source](http://bioinformaticstools.mayo.edu/research/patterncnv/)|
|[pyloh](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#pyloh)|2014|UC Irvine|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103592/)|[source](https://github.com/uci-cbcl/PyLOH)|
|[qdnaseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#gdnaseq)|2014|VU University Medical Center|[study](http://genome.cshlp.org/content/early/2014/09/18/gr.175141.114)|[source](http://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html)|
|[cnvem](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvem)|2013|UCLA|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590897/)|[source](http://genetics.cs.ucla.edu/cnvem/index.html)|
|[excavator](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#excavator)|2013|U Florence|[study](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r120)|[source](https://sourceforge.net/projects/excavatortool/)|
|[fishingcnv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#fishingcnv)|2013|McGill U Canada, Majewski|[study](https://bioinformatics.oxfordjournals.org/content/29/11/1461)|[source](https://sourceforge.net/projects/fishingcnv/)|
|[matchclip](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#matchclip)|2013|U Pennsylvania, Philadelphia|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3744852/)|[source](https://github.com/yhwu/matchclips2)|
|[oncosnpseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#oncosnpseq)|2013|Imperial College London|[study](https://academic.oup.com/bioinformatics/article/29/19/2482/187976)|[source](https://sites.google.com/site/oncosnpseq/)|
|[patchwork](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#patchwork)|2013|Upsala University, Sweden|[study](https://www.ncbi.nlm.nih.gov/pubmed/23531354/)|[source](http://patchwork.r-forge.r-project.org/)|
|[theta2](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#theta2)|2013|Brown, Raphael|[study](http://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-7-r80)|[source](http://compbio.cs.brown.edu/software/)|
|[absolute](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#absolute)|2012|Broad, Getz|[study](http://www.nature.com/nbt/journal/v30/n5/full/nbt.2203.html)|[source](http://archive.broadinstitute.org/cancer/cga/absolute)|
|[apolloh](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#apolloh)|2012|British Colombia Cancer Agency|[study](https://www.ncbi.nlm.nih.gov/pubmed/22637570)|[source](http://compbio.bccrc.ca/software/apolloh/)|
|[cnanorm](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnanorm)|2012|U Leeds|[study](https://www.ncbi.nlm.nih.gov/pubmed/22039209)|[source](http://www.precancer.leeds.ac.uk/software-and-datasets/cnanorm/)|
|[cnvhitseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#ncvhitseq)|2012|Imperial College London|[study](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-12-r120)|[source](https://sourceforge.net/projects/cnvhitseq/)|
|[conifer](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#conifer)|2012|U Wash Seattle|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3409265/)|[source](http://conifer.sourceforge.net/)|
|[contra](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#contra)|2012|Peter MacCallum Cancer Centre|[study](https://www.ncbi.nlm.nih.gov/pubmed/22474122)|[source](https://sourceforge.net/projects/contra-cnv/)|
|[cops](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cops)|2012|Institute of Applied Bioinformatics, Bangalore India|[study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0047812)|[source](https://github.com/binaypanda/COPS)|
|[erds](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#erds)|2012|Duke University|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3511991/)|N/A|
|[exomedepth](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#exomedepth)|2012|U Cambridge, Negentsev|[study](http://bioinformatics.oxfordjournals.org/content/28/21/2747.long)|[source](https://cran.r-project.org/web/packages/ExomeDepth/index.html)|
|[magnolya](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#magnolya)|2012|Netherlands Bioinformatics Centre|[study](https://www.ncbi.nlm.nih.gov/pubmed/23047563)|[source](https://sourceforge.net/projects/magnolya/)|
|[seqcbs](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#seqcbs)|2012|Stanford University|[study](https://arxiv.org/abs/1206.6627)|[source](https://cran.r-project.org/web/packages/seqCBS/index.html)|
|[xhmm](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#xhmm)|2012|Mt Sinai, Purcell|[study](http://www.cell.com/AJHG/abstract/S0002-9297%2812%2900417-X)|[source](http://atgu.mgh.harvard.edu/xhmm/index.shtml)|
|[bic-seq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#bic-seq)|2011|Harvard|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219132/)|[source](http://compbio.med.harvard.edu/Supplements/PNAS11.html)|
|[cnvnator](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnvnator)|2011|Yale, Gerstein|[study](https://www.ncbi.nlm.nih.gov/pubmed/21324876)|[source](https://github.com/abyzovlab/CNVnator)|
|[exomecnv](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#exomecnv)|2011|Dana-Farber Cancer Institute|[study](http://bioinformatics.oxfordjournals.org/content/27/19/2648.short)|[source](https://cran.r-project.org/src/contrib/Archive/ExomeCNV/)|
|[exomecopy](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#exomecopy)|2011|U Oslo|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517018/)|[source](http://www.bioconductor.org/packages/release/bioc/html/exomeCopy.html)|
|[jointslm](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#jointslm)|2011|Careggi Hospital, Italy|[study](https://www.ncbi.nlm.nih.gov/pubmed/21321017/)|[source](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105418/bin/supp_39_10_e65__index.html)|
|[readdepth](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#readdepth)|2011|Baylor College of Medicine|[study](https://www.ncbi.nlm.nih.gov/pubmed/21305028/)|[source](https://github.com/chrisamiller/readDepth)|
|[cnaseg](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnaseg)|2010|Li Ka Shing Centre, UK|[study](https://www.ncbi.nlm.nih.gov/pubmed/20966003/)|[source](http://www.compbio.group.cam.ac.uk/software/cnaseg/)|
|[cnd](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnd)|2010|Sanger Institute|[study](https://academic.oup.com/bioinformatics/article/26/4/565/243503)|[source](http://www.sanger.ac.uk/resources/software/cnd.html)x|
|[cnver](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnver)|2010|U Toronto|[study](docs.google.com/document/d/1Ki36Ye7cDDA6nlEhgvICsg_IsSguxyBb1u4MCWi5so0/edit)|[source](http://compbio.cs.toronto.edu/CNVer/)|
|[copyseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#copyseq)|2010|EMBL|[study](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000988)|[source](http://www.mybiosoftware.com/copyseq-1-7-1-discovery-copy-number-genotyping-genomic-structural-variants.html)|
|[freec](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#freec)|2010|Institut Curie, Barillot|[study](http://bioinformatics.oxfordjournals.org/content/27/2/268.short)|[source](http://boevalab.com/FREEC/)|
|[novelseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#novelseq)|2010|Simon Fraser University, Canada|[study](https://www.ncbi.nlm.nih.gov/pubmed/20385726)|[source](http://novelseq.sourceforge.net/Home)|
|[rsw-seq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#rsw-seq)|2010|Harvard Medical School, Park lab|[study](https://www.ncbi.nlm.nih.gov/pubmed/20718989/)|[source](http://compbio.med.harvard.edu/Supplements/BMCBioinfo10-2.html)|
|[cmds](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cmds)|2009|WashU St Louis, Province|[study](http://bioinformatics.oxfordjournals.org/content/26/4/464.full)|[source](https://dsgweb.wustl.edu/qunyuan/software/cmds/)|
|[cnv-seq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#cnv-seq)|2009|National University of Singapore|[study](https://www.ncbi.nlm.nih.gov/pubmed/19267900/)|[source](http://tiger.dbs.nus.edu.sg/CNV-seq/)|
|[mrfast](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#mrfast)|2009|Howard Hughes Medical Institute|[study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2875196/)|[source](https://sourceforge.net/projects/mrfast/)|
|[rdxplorer](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#rdxplorer)|2009|Cold Spring Harbor|[study](https://www.ncbi.nlm.nih.gov/pubmed/19657104)|[source](http://rdxplorer.sourceforge.net/)|
|[segseq](https://github.com/deaconjs/ThousandVariantCallersRepo/wiki/CNV#segseq)|2009|Broad Institute|[study](https://www.ncbi.nlm.nih.gov/pubmed/19043412/)||


### bic-seq2

Notes: somatic calls. 

### clamms

Notes: exome input. exome capture data, normalizes GC content

Algorithm: HMM/mixture model

Description: Copy number estimation using Lattice-Aligned Mixture Models. Evaluate the adherence of CNV calls from CLAMMS and four other algorithms to Mendelian inheritance patterns on a pedigree


### cnvkit

Notes: targeted sequencing input. somatic calls. targeted reads, uses off-target reads

Used by: biocondor

Algorithm: CBS algorithm (circular binary segmentation)

Description: CNV detection that takes advantage of both on– and off-target sequencing reads and applies a series of corrections to improve accuracy in copy number calling.


### erds-pe

Algorithm: paired HMM


### ioncopy

Notes: for panel/amplicon, tumor population only; no normal controls used

Algorithm: amplicon read depth population statistics calling from tumor-only cohort. 

Description: estimate a null distribution of copy numbers using outlier-robust statistics and assess the significance of CNAs by comparison with this null distribution. In this way, p-values are obtained for each amplicon in each tumor that are subsequently corrected for multiple hypothesis testing. ... For all simulated situations, CN gains of 5 and more can be detected with high sensitivity and specificity. Detection of CN gains of 4 is feasible in some situations, for example when the number of genes under investigation is low.

### popsv

Notes: population-based calls. 

Description: Population-based detection of structural variation from High-Throughput Sequencing


### srbreak

Algorithm: read-depth first for CNV-region detection, followed by split read analysis to locate breakpoints.

Compared to: Pindel, DELLY, MATCHCLIP, SoftSearch, CNVnator

Description: combines a read-depth-based approach and a split-read-based approach to identify breakpoints for different duplication/deletion events inside a large CNVR. The strength of this pipeline comes through its use of multiple samples in one CNV genotype group to identify common breakpoints for that group. It is able to use both single-end and paired-end reads from HTS data.


### triocnv

Notes: trio calls. 

### as-genseng

Notes: combines allele-specific RC with total RC

Algorithm: HMM/model total and allele specific separate

Description: incorporates allele-specific read counts in CNV detection and estimates ASCN using either WGS or WES data


### codex

Notes: exome only. population-based calls. includes terms that specifically remove biases due to GC content, exon length and capture and amplification efficiency, and latent systematic artifacts

Algorithm: Poisson latent factor/recursive segmentation

Description: relies on the availability of multiple samples processed using the same sequencing pipeline. Unlike current approaches, CODEX uses a Poisson log-linear model that is more suitable for discrete count data. The normalization model in CODEX includes terms that specifically remove biases due to GC content, exon length and capture and amplification efficiency, and latent systematic artifacts


### conserting

Notes: Produces CNV/SV calls. dep on bambino, picard

Algorithm: regression tree segmentation

Description: integrate read-depth change with structural variation (SV) identification through an iterative process of segmentation by read depth, segment merging, and localized SV detection. recursive partitioning techniques to find the transition point for read depth changes.


### copywriter

Notes: for targeted sequencing. reference-free

Algorithm: CBS algorithm, uses off-target sequencing reads

Description: exploiting ‘off-target’ sequence reads. CopywriteR allows for extracting uniformly distributed copy number information, can be used without reference, and can be applied to sequencing data obtained from various techniques including chromatin immunoprecipitation and target enrichment on small gene panels


### falcon

Notes: somatic calls. calculates allele-specific copy numbers, deduce clonal history

Algorithm: bivariate mixed Binomial,  Bayesian criterion for count estimates

Description: based on a change-point model on a bivariate mixed Binomial process, which explicitly models the copy numbers of the two chromosome haplotypes and corrects for local allele-specific coverage biases. By using the Binomial distribution rather than a normal approximation, falcon more effectively pools evidence from sites with low coverage


### grom-rd

Notes: wgs, no control req

Description: excessive coverage masking, GC bias mean and variance normalization, GC weighting, dinucleotide repeat bias detection and adjustment, and a size-varying sliding window CNV search.


### modsara

Algorithm: screening and ranking algo


### abscn-seq

Notes: estimates purity & ploidy. exome only. 


### adtex

Notes: estimates purity and ploidy. exome only. testing somatic calling.

Algorithm: HMM

Description: uses two Hidden Markov Models to predict copy number and genotypes and computationally resolves polyploidy/aneuploidy, normal cell contamination and signal baseline shift. Our method makes explicit detection on chromosome arm level events, which are commonly found in tumour samples


### canoes

Notes: exome only. 

Algorithm: models sequence coverage using the negative binomial distribution


### Climat

Notes: estimates LOH. robust to contamination and aneuploidy

Description: takes integrated analysis of read count and allele frequency derived from sequenced tumor samples, and provides extensive data processing procedures including GC-content and mappability correction of read count and quantile nor-malization of B allele frequency


### cnvcapseq

Notes: targeted resequencing input. 

Description: cnvCapSeq integrates evidence from both RD and read pairs (RP) to achieve high breakpoint resolution regardless of coverage uniformity


### cnvoffseq

Description: normalization framework for off-target read depth that is based on local adaptive singular value decomposition (SVD). This method is designed to address the heterogeneity of the underlying data and allows for accurate and precise CNV detection and genotyping in off-target regions.


### cnvrd2

Validated vs: cnvnator, cn.mops

Description: first uses observed read-count ratios to refine segmentation results in one population. Then a linear regression model is applied to adjust the results across multiple populations, in combination with a Bayesian normal mixture model to cluster segmentation scores into groups for individual CN counts.


### m-hmm

Algorithm: HMM


### oncocnv

Notes: amplicon input. 

Description: defining a method to normalize read coverage with a small set of normal control samples and (ii) assigning statistical significance to putative CNAs resulting from the segmentation of normalized profiles

### patterncnv

Notes: exome only. somatic calls. WIG format bams for speed

Algorithm: compares paired samples

Description: accounts for the read coverage variations between exons while leveraging the consistencies of this variability across different samples


### pyloh

Notes: estimates LOH, purity, ploidy.

Used by: biocondor

Description: deconvolve read mixture to identify reads associated with tumor cells or a particular subclone of tumor cells. Integrate somatic copy number alterations and loss of heterozygosity in a unified probabilistic framework.


### qdnaseq

Notes: shallow depth ok, robust to FFPE

Algorithm: read-depth, no paired analysis needed

### cnvem

Description: use maximum likelihood to estimate locations and copy numbers of copied regions and implement an expectation-maximization (EM) algorithm


### convex

Notes: exome only. 

Algorithm: HMM

Description: uses ratio of tumour and matched normal average read depths at each exonic region, to predict the copy gain or loss


### excavator

Notes: exome only. efficient processing

Algorithm: read-count/HMM based with 3-step normalization, segmentation

Description: combines a three-step normalization procedure with a novel heterogeneous hidden Markov model algorithm and a calling method that classifies genomic regions into five copy number state


### fishingcnv

Description: compares coverage depth in a test sample against a background distribution of control samples and uses principal component analysis to remove batch effects


### matchclip

Description: Our method searches for reads that potentially span the breakpoints of a CNV by screening CIGAR strings. If a long S part is at the 3′(right)-side, we can use its alignment to determine the 5′(left)-side of the breakpoint, and vice versa. Our method searches for two reads that span the same CNV with the long soft-clipped parts at the either end in order to locate both breakpoints of the CNV. To ensure the two reads indeed cover the same CNV, we require that they overlap in a certain orientation and their common string includes both of the soft-clipped parts.

### oncosnpseq

Algorithm: mixed Binomial model for multiple tumor genotypes contaminated with normal cells, HMM resolves most likely set of mixtures.


### patchwork

Notes: uses WGS input.


### theta

Notes: estimates purity & ploidy. somatic calls. estimates purplo, efficient

Used by: biocondor

Algorithm: maximum likelihood mixture decomposition problem

Description: infers the most likely collection of genomes and their proportions in a sample, for the case where copy number aberrations distinguish subpopulations. THetA successfully estimates normal admixture and recovers clonal and subclonal copy number aberrations


### absolute

Notes: estimates LOH. 

Algorithm: estimates loss of heterozygosity

Description: detect subclonal heterogeneity and somatic homozygosity, and it can calculate statistical sensitivity for detection of specific aberrations


### apolloh

Notes: estimates LOH. somatic calls. 

Algorithm: estimates loss of heterozygosity. 

Description: a hidden Markov model (HMM) for predicting somatic loss of heterozygosity and allelic imbalance in whole tumour genome sequencing data.


### cnanorm

Description: identify the multi-modality of the distribution of smoothed ratios. Then we use the estimates of the mean (modes) to identify underlying ploidy and the contamination level, and finally we perform the correction.


### cnanorm

Notes: somatic calls.


### cnvhitseq

Description: jointly models evidence from RD, RPs and SRs at the population level. pool information across individual samples and reconcile copy number differences among data sources


### conifer

Notes: exome only. population-based calls. 

Description: this method can be used to reliably predict (94% overall precision) both de novo and inherited rare CNVs involving three or more consecutive exons


### contra

Notes: exome only. 

Algorithm: CBS algorithm

Description: calls copy number gains and losses for each target region based on normalized depth of coverage. Our key strategies include the use of base-level log-ratios to remove GC-content bias, correction for an imbalanced library size effect on log-ratios, and the estimation of log-ratio variations via binning and interpolation


### erds

Notes: available?

Description: starts from read depth (RD) information, and integrates other information including paired end mapping (PEM) and soft-clip signature to call CNVS


### exomedepth

Notes: exome only. de novo? mendelian

Used by: biocondor

Description: Calls copy number variants (CNVs) from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.


### magnolya

Notes: reference-free. 

Description: enables copy number variation (CNV) detections without using a reference genome. Magnolya directly compares two next-generation sequencing datasets.


### seqcbs

Description: based on a simple and flexible inhomogeneous Poisson
Process model for sequenced reads. We derive the score and generalized likelihood
ratio statistics for this model to detect regions where the read intensity shifts
in the target sample, as compared to a reference. We construct a modified Bayes
information criterion (mBIC) to select the appropriate number of change points
and propose Bayesian point-wise confidence intervals as a way to assess the confi-
dence in the copy number estimates.

### xhmm

Notes: exome only. 

Algorithm: PCA/HMM

Description: uses principal component analysis (PCA) normalization and a hidden Markov model (HMM) to detect and genotype copy number variation (CNV) from normalized read-depth data from targeted sequencing experiments.


### bic-seq

Notes: somatic

Description: Combines normalization of the data at the nucleotide level and Bayesian information criterion-based segmentation to detect both somatic and germline copy number variations


### cmvnator

Notes: no control req

Used by: metasv

Algorithm: read coverage

Description: CNVnator is able to discover CNVs in a vast range of sizes, from a few hundred bases to megabases in length


### exomecnv

Notes: exome only. 

Description: a statistical method to detect CNV and LOH using depth-of-coverage and B-allele frequencies, from mapped short sequence reads


### exomecopy

Notes: exome only. 

Description: an HMM for predicting copy number state in exome and other targeted sequencing data using observed read counts and positional covariates


### jointslm

Notes: population-based.

### readdepth

Notes: no control req

### cnaseg

Notes: somatic calls. 


### cnver

Description: supplements the depth-of-coverage with paired-end mapping information, where mate pairs mapping discordantly to the reference serve to indicate the presence of variation.


### copyseq

### freec

Notes: no control req

Description: The tool deals with two frequent problems in the analysis of cancer deep-sequencing data: absence of control sample and possible polyploidy of cancer cells.


### novelseq

Description: discover the content and location of long novel sequence insertions


### rsw-seq

Notes: somatic calls. 


### cmds

Description: correlation matrix diagonal segmentation (CMDS), identifies RCNAs based on a between-chromosomal-site correlation analysis.

### cnv-seq

Notes: somatic calls


### rdxplorer

Notes: no contol req

Description: copy number variants (CNV) detection in whole human genome sequence data using read depth (RD) coverage. CNV detection is based on the Event-Wise Testing (EWT) algorithm


### segseq

Notes: somatic calls
